packages/base-zone/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@agoric/base-zone",
+  "version": "1.0.0",
```

Comment:
Please use a 0 major version to postpone the breaking change discipline required by semver.